walkthroughs/calculation.md

```diff
@@ -74,7 +74,7 @@ Next, change into the Goodie repository's home directory, `zeroclickinfo-goodies
 [01:07 PM codio@border-carlo workspace ]$ cd zeroclickinfo-goodies
 ```
 
-Before doing any coding, it's recommended you work from a [new, separate Git branch](http://docs.duckduckhack.com/resources/git-workflow.html#step-3-create-a-working-branch-for-coding) *and not master*. A separate branch allows you to keep your repository up-to-date, and work on multiple pull requests at one time.
+Before doing any coding, it's recommended you work from a [new, separate Git branch](http://docs.duckduckhack.com/resources/git-workflow.html#step-3-create-a-working-branch-for-coding) *and not master*. A separate branch allows you to keep your repository up-to-date, and work on multiple pull requests at one time. Also, make sure that you make the new branches from the master branch only.
```

Comment:
This looks all good, feels quite wordy. Perhaps 'Make sure that the branch is a fork of the DuckDuckGo master, and not any other branch' as it avoids duplicating words, while also being specific on what master branch to use.